app/views/notes/rss.rss.builder

```diff
@@ -8,15 +8,15 @@ xml.rss :version => "2.0" do
    @notes.each do |node|
 
      body = node.latest.render_body
-     body = "<p><img src='"+node.main_image.path(:default)+"'/></p> " + node.body if node.main_image
+     body = "<p><![CDATA[ <img src='"+node.main_images.path(:default)+"' alt='"+node.main_image.title+"' > ]]></p> " + node.body if node.main_image
+     author = node.author.mail+" ("+node.author.name+")"
```

Comment:
This is looking great -- I think @sagarpreet-chadha mentioned this line should actually be down on the `xml.author` field on line 15/16 -- can you move it there? And rather than the email, we can use the username, so `node.author.username`.

In fact, we want to try to use their twitter account name if they have one. So we could try using this feature: https://github.com/publiclab/plots2/blob/master/app/models/user.rb#L151

```ruby
xml.author (node.author.has_power_tag('twitter') ? node.author.get_value_of_power_tag('twitter') : node.author.username)
```

That way if there is a twitter tag attached, we use it, but if not, we default to the username. Make sense?

Reply:
and thank you!!!!!